test/test_evaluate/test_docstring.py

```diff
@@ -1,13 +1,20 @@
 """
 Testing of docstring related issues and especially ``jedi.docstrings``.
+
+CommandLine:
+    # Switch to the right directory
+    cd $(python -c "import jedi; from os.path import dirname, abspath; print(dirname(dirname(abspath(jedi.__file__))))")
+    # Run the doctests in this module
+    tox -e py test/test_evaluate/test_docstring.py
+
```

Comment:
I don't think you need to mention that here. What you can do is put something like this into the docs, but especially the second command is general knowledge (you can find that in every tox tutorial). It's even easier if you use pytest. The cd command is also not extremely useful, since most people tend to just go to that place (in my case $HOME/programming/jedi).